src/classlibnative/bcltype/decimal.cpp

```diff
@@ -399,19 +399,19 @@ unsigned int DecDivMod1E9(DECIMAL* value)
     LIMITED_METHOD_CONTRACT
 
     _asm {
-        mov     ebx,value
-        mov     ecx,1000000000
-        xor     edx,edx
-        mov     eax,[ebx+4]
-        div     ecx
-        mov     [ebx+4],eax
-        mov     eax,[ebx+12]
-        div     ecx
-        mov     [ebx+12],eax
-        mov     eax,[ebx+8]
-        div     ecx
-        mov     [ebx+8],eax
-        mov     eax,edx
+        MOV     ebx,value
```

Comment:
Could you please revert the instruction name case changes? We use lower case in all the asm code. I hope the new compiler isn't comaplaining about the case, that would be unfortunate.